### ISSUE TITLE
Why Snikket?

### ISSUE THREAD

**Issue by solvaholic** [winans-fam/snikket#1]
tl;dr?

https://snikket.org

> Everything you expect from an easy-to-use messaging platform, but now under your control.

> The Snikket network is instead made of lots of smaller independent services that can all communicate with each other.

- - - - 

Why Snikket? ProsodyIM's [guide](https://prosody.im/download/start) sent me to Snikket:

> 💡 Tip: New to XMPP? Looking for a simple preconfigured complete messaging server you can run in Docker? Take a look at our sister project, [Snikket](https://snikket.org/)!

Why was I looking at [ProsodyIM](https://prosody.im)? I want to host a federated chat server for winans.net such that I can run an instance on my home network and an instance on the internet. This way the private instance can be accessed via the public instance, and when it's isolated from the internet it's still available to host local chat. (At least: That's the idea. We'll see how it plays out.)

Why was I looking for local chat that can run without internet access? We're hosting our own DNS, monitoring, ebook library, and other things. When this stuff loses its internet connection I want to be notified; And I want to be able to communicate with other users on our home network about status updates and stuff.

External dependencies like Amazon Web Services, PagerDuty, Twilio, and hosted identity services all fail when the internet's unavailable. Cellular services provide another option (think SMS and 3G/4G/LTE data) but also require external dependencies.

But .. why ProsodyIM as opposed to, say, matrix.org or Mastodon or Rocket.Chat? [XMPP](https://xmpp.org). Other self-host options seem esoteric, lack federation, require external dependencies, or lack community-supported client software. XMPP is well-defined, straightforward, and common.
